README.md

```diff
@@ -25,64 +25,93 @@ What this `kubectl` plugin does is glue your kubectl config to a common password
 
 ## Installation
 
-### Mac OS X
+### Mac OS X/Unix/Linux
 
 ```bash
-TODO:
+go install github.com/chrisns/kubectl-passman
```

Comment:
download binary from releases on Github?

Reply:
even better... this should be a kubectl plugin for krew

Reply:
binary creation is a todo :)
krew, yes could be, will add a todo